modules/geolocation/geolocation.py

```diff
@@ -0,0 +1,68 @@
+import numpy as np
+
+class Geolocation:
+    """Locates the geographical position of a set of pixels"""
+
+    # TODO Class members
+    def __init__(self):
```

Comment:
Create a constructor param for the array of points to convert to geocordinates, set pixelsToGeographicals (with new name) to that

Reply:
I thought we were re-using the class to save having to constantly do constructor and destructor calls? Also, pixelsToGeographicals will be populated by the output of the gather_point_pairs() function (currently a placeholder).